frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboard.tsx

```diff
@@ -209,111 +165,39 @@ class PublicOrEmbeddedDashboardInner extends Component<PublicOrEmbeddedDashboard
       hideParameters,
       navigateToNewCardFromDashboard,
       selectedTabId,
+      setParameterValue,
+      slowCards,
+      dashboardId,
+      cardTitled,
     } = this.props;
 
-    const buttons = !isWithinIframe()
-      ? getDashboardActions({
-          dashboard,
-          hasNightModeToggle,
-          isFullscreen,
-          isNightMode,
-          onFullscreenChange,
-          onNightModeChange,
-          onRefreshPeriodChange,
-          refreshPeriod,
-          setRefreshElapsedHook,
-          isPublic: true,
-        })
-      : [];
-
-    const visibleDashcards = (dashboard?.dashcards ?? []).filter(
-      dashcard => !isActionDashCard(dashcard),
-    );
-
-    const dashboardHasCards = dashboard && visibleDashcards.length > 0;
-
-    const tabHasCards =
-      visibleDashcards.filter(
-        (dc: DashboardCard) => dc.dashboard_tab_id === selectedTabId,
-      ).length > 0;
-
     return (
-      <EmbedFrame
-        name={dashboard && dashboard.name}
-        description={dashboard && dashboard.description}
+      <PublicOrEmbeddedDashboardView
```

Comment:
I didn't decide how to sort props. VS code refactor does that and I don't want to bother with reodering them.